src/test/java/com/passbook/sparkeighteen/service/UserServiceTest.java

```diff
@@ -162,5 +162,35 @@ public void inValidPayload_passwordMismatch_loginUnSuccessful() throws Exception
 
     }
 
+    @Test
+    public void missingUser_deleteProfileUnSuccessful() throws Exception {
+        Integer userID = 1;
+
+        when(userRepository.findById(userID)).thenReturn(Optional.empty());
+
+        String response = userService.deleteProfile(userID);
+        assertEquals("user id is not found", response);
+
+    }
+
+    @Test
+    public void userExists_deleteProfileSuccessful() throws Exception {
```

Comment:
Not following the pattern
https://enterprisecraftsmanship.com/posts/you-naming-tests-wrong/
[MethodUnderTest]_[Scenario]_[ExpectedResult]
Do it for all places

Reply:
read the block, it was helpful